### ISSUE TITLE
about architecture

### ISSUE THREAD

**Issue by TaekyungKi** [wyhsirius/LIA#13]
Hi, thanks for the nice work. I have some questions about architecture.
1. In Fig.8 in, toRGB of shape 4 x 4, 512 not is used (refer to the red box in the below image), right? This is implemented by "self.to_rgb1=ToRGB(...)" in styledecoder.py, but didn't used in forward pass and also in backward pass. (This yields unused parameter error when training). I'm not sure this is intended implementation. I think this layer is important because it is the starting layer and the only the un-warped feature, which may helps to preserve id.

<img width="474" alt="스크린샷 2022-09-25 오전 1 12 19" src="https://user-images.githubusercontent.com/55676509/192108140-636c7bca-8710-4b4b-81ea-67b30ccdf522.png">

2. Did you use NoiseInjection in Synthesis network? The parameters are assigned when initializing the NoiseInjection, but it is not used in the model.

If I missed something, please free to tell. Anyway, I am enjoying LIA. Thanks.

**Comment by TaekyungKi** [comment]
<img width="843" alt="dd" src="https://user-images.githubusercontent.com/55676509/193198875-c20e7bf3-177f-40de-a3a0-1b8a8a25763a.png">


i also found some other architecture difference not reported in the paper and i denote them on above figure.
1. The attention mask "sigma" is also used as "1-sigma" for weight sum with the input. This is reasonable but not reported.
2. There is additional skip layer which comes from the previous layer. As it comes the previous layer, it is upsampled x2 and added to the feature map which is output of conv 3x3 of ToFlow block. Can you explain this is right?

Thanks.
